proto/desmos/posts/v1beta1/posts.proto

```diff
@@ -71,6 +71,22 @@ message CommentIDs { repeated string ids = 1; }
 
 // ___________________________________________________________________________________________________________________
 
+// CommentsState contains all the possible comments states
+enum CommentsState {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  // COMMENTS_STATE_UNSPECIFIED
+    COMMENTS_STATE_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "Unspecified"];
+
+  // COMMENT_STATE_ALLOWED tells that's possible to comment a post
+  COMMENTS_STATE_ALLOWED = 1 [(gogoproto.enumvalue_customname) = "Allowed"];
+
+  // COMMENT_STATE_BLOCKED tells that's not possible to comment a post
+  COMMENTS_STATE_BLOCKED = 2 [(gogoproto.enumvalue_customname) = "Blocked"];
```

Comment:
I think it would be much more clear to have `CommentsStateUnspecified`, `CommentsStateAllowed` and `CommentsStateBlocked` as values. This would allow us to not get confused with other enum values: 

```suggestion
  // COMMENTS_STATE_UNSPECIFIED
  COMMENTS_STATE_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "CommentsStateUnspecified"];

  // COMMENT_STATE_ALLOWED tells that's possible to comment a post
  COMMENTS_STATE_ALLOWED = 1 [(gogoproto.enumvalue_customname) = "CommentsStateAllowed"];

  // COMMENT_STATE_BLOCKED tells that's not possible to comment a post
  COMMENTS_STATE_BLOCKED = 2 [(gogoproto.enumvalue_customname) = "CommentsStateBlocked"];
```